package.json

```diff
@@ -109,19 +87,13 @@
     "styled-components": "^4.0.0",
     "webpack": "^4.34.0"
   },
-  "peerDependencies": {
-    "date-fns": "1.30.1",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.1",
-    "styled-components": "^4.0.0"
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged && yarn run lint"
```

Comment:
linter is going to make the pre-commit hook a lot longer, are we sure we want to force this for each commit ?

Reply:
It was already like this, I didn't change anything